source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -736,15 +739,19 @@ module Error = struct
   let pp_kind formatter = function
     | FileNotFound -> Format.fprintf formatter "File not found"
     | FileRead -> Format.fprintf formatter "Could not read file"
-    | InvalidJson error -> Format.fprintf formatter "%s" error
+    | InvalidJson { message; location } ->
+        Format.fprintf
+          formatter
+          "%s lines `%d` to `%d`, columns `%d` to `%d`."
```

Comment:
Could we stick to the traditional format `<path>:<start_line>:<start_column>: <message>`?

Reply:
yes